feature-libs/asm/root/model/bind-cart.model.ts

```diff
@@ -0,0 +1,4 @@
+export interface BindCartOptions {
```

Comment:
These are more parameters than options in this case.  Should we name the interface `BindCartParams` `BindCartParameters`?  What do you think?